src/merkle/path.rs

```diff
@@ -1,6 +1,12 @@
 use super::{vec, NodeIndex, Rpo256, Vec, Word};
 use core::ops::{Deref, DerefMut};
 
+#[cfg(not(feature = "std"))]
+pub use alloc::slice;
+
+#[cfg(feature = "std")]
+pub use std::slice;
```

Comment:
We can require `alloc::slice` directly as the compilation should replace it by `std::slice`, if `std` is available

Reply:
For me that fails with:

```
error[E0432]: unresolved import `alloc`
 --> src/merkle/path.rs:4:9
  |
4 | pub use alloc::slice;
  |         ^^^^^ help: a similar path exists: `std::alloc`

For more information about this error, try `rustc --explain E0432`.
```